templates/chat/parts/card-activities.hbs

```diff
@@ -0,0 +1,16 @@
+<section class="activities">
+    <strong class="roboto-condensed-upper">{{ localize "DND5E.CHATMESSAGE.Activities" }}</strong>
+    <ul class="unlist">
+        {{#each activities}}
+        <li class="activity flexrow item-tooltip" data-activity-uuid="{{ uuid }}" data-item-uuid="{{ item.uuid }}">
```

Comment:
Wasn't seeing these tooltips activating when I tested. Do we need additional code in `ChatMessage5e` to activate them?